scenes/game_elements/props/eternal_loom/components/eternal_loom_interaction.dialogue

```diff
@@ -1,13 +1,21 @@
 # SPDX-FileCopyrightText: The Threadbare Authors
 # SPDX-License-Identifier: MPL-2.0
-~ start
+~ blurb
```

Comment:
I had to look up if `blurb` meant something 😅 , is it referring to this?:
https://dictionary.cambridge.org/es/diccionario/ingles/blurb

Like, in this context, a short introduction?

Reply:
Great point. "Introduction" would be a better name (and certainly a less obscure word)

Reply:
I went with "welcome"